recipes/lapack95/make.inc

```diff
@@ -0,0 +1,47 @@
+#
+#  -- LAPACK95 interface driver routine (version 2.0) --
+#     UNI-C, Denmark; Univ. of Tennessee, USA; NAG Ltd., UK
+#     August 5, 2000
+#
+FC_fr	 = $(FC) -ffree-form
+FC_ff    = $(FC) -ffixed-form
+# -dcfuns  Enable recognition of non-standard double
+#          precision  complex intrinsic functions
+# -dusty   Allows the compilation and execution of "legacy"
+#          software by downgrading the category  of  common
+#          errors  found  in  such software from "Error" to
+# -ieee=full enables all IEEE arithmetic facilities
+#          including non-stop arithmetic.
+
+OPTS0    = -pipe
+MODLIB   = -I$(PREFIX)/include
+OPTS1    = -c -O3 $(OPTS0)
+OPTS3    = $(OPTS1) $(MODLIB)
+OPTL     = -o 
+OPTLIB   =
+
+LAPACK_PATH = $(PREFIX)/lib
+
+LAPACK95 = ../lapack95.a
+LAPACK77 = $(LAPACK_PATH)/liblapack.a
+TMG77    = $(LAPACK_PATH)/libtmglib.a
+BLAS     = $(LAPACK_PATH)/libblas.a
```

Comment:
Is it possible to produce dynamic libraries instead?

Reply:
I will try to take a look into it. And get back to you on that. Should I look into provide both?

Reply:
Ok thanks. Just the dynamic libraries are fine

Maintainers can optionally provide static libraries in a separate package, but that’s not needed

This CFEP is relevant

https://github.com/conda-forge/cfep/blob/master/cfep-18.md

Reply:
I looked over the instructions provided with the package there are no existing ways to build a dynamic version. Can we go with the static one? I can try later to patch their build system and try to get a shared version.

Reply:
This is not creating a static library. Here you are using the static versions of the dependencies. Please use the dynamic libraries of dependencies

Reply:
@isuruf it creates `lapack95.a` (static) and that is hardcoded all over the makefile for this library.
I can change the dependencies to be shared and see how that works.

Reply:
That's not what I was talking about. For eg:  `BLAS     = $(LAPACK_PATH)/libblas.a` uses `libblas.a` dependency from `libblas` package in conda-forge.

Reply:
@isuruf sorry for the misunderstanding. Addressed via https://github.com/conda-forge/staged-recipes/pull/14212/commits/cfe495c7a35765a0cfda06f19aee443b0b6bbfdd